content/en/blog/2024/collecting-otel-compliant-java-logs-from-files/index.md

```diff
@@ -0,0 +1,383 @@
+---
+title: Collecting OpenTelemetry-compliant Java logs from files
+linkTitle: OTel-compliant Java logs from files
+date:
```

Comment:
```suggestion
date: 2024-12-02
```

please keep this open until we have set the exact date.

I would like to wait for @chalin to re-review and since this week most people are out due to Thanksgiving in the US I would argue that publishing next week is better anyways

Reply:
```suggestion
date: 2024-12-03
```